Source/CarthageKit/BinaryProject.swift

```diff
@@ -1,16 +1,19 @@
 import Foundation
 import Result
 
+/// The binary project.
```

Comment:
Is this documentation comment required by a rule?

In general, we greatly appreciate adding documentation in PRs, but this particular one requires more in-depth details. (Also, with Carthage managing not-just-a-single `BinaryProject`, the comment’s `The` might be misleading).

Opening a issue requesting documentation for `BinaryProject` would be great, if you'd like to file that 😄